### PR TITLE
Feat/http connection

### PR DESCRIPTION
**Pull request by 8ball030**
Adds in the http connection necessary for the front end to retrieve the data.

We eject this from the Fetchai repo so that we are able to enable corse and server multiple servers

### CODE REVIEW

File: agents/packages/ethlisbon/skills/price_prophet/behaviours.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def get_strict(self, key: str) -> Any:
         """Get strict from SynchronizedData"""
         return self.synchronized_data.db.get_strict(key)
 
-    @functools.cached_property
+    @property
```
Comment (Karrenbelt) [inline agents/packages/ethlisbon/skills/price_prophet/behaviours.py:102]: why the change?

**Review by Karrenbelt — APPROVED** [review]

File: agents/packages/ethlisbon/skills/price_prophet/behaviours.py (hunk 1 above)
Comment (8ball030) [reply agents/packages/ethlisbon/skills/price_prophet/behaviours.py:102]: for some reason this worked but it didnt previously
